File: VolleyballApp/VolleyballApp.API/Models/Team.cs
using System;
using System.Collections.Generic;

namespace VolleyballApp.API.Models
{
    public class Team
    {
        public int Id { get; set; }
        public string TeamName { get; set; }
        public int OwnerId { get; set; }
        public User Owner { get; set; }
        public ICollection<User> Users { get; set; }
        public DateTime DateCreated { get; set; }
        public string Description { get; set; }
        public Photo Photo { get; set; }
        public int RankingPoints { get; set; }
    }
}

File: VolleyballApp/VolleyballApp-SPA/src/app/_models/team.ts
import { League } from './league';
import { Photo } from './photo';
import { User } from './user';
import { UserTeam } from './userTeam';

export interface Team {
    id: number;
    teamName: string;
    owner?: User;
    users?: User[];
    userTeams?: UserTeam[];
    dateCreated?: Date;
    description?: string;
    photo?: Photo;
    photoUrl?: string;
    league?: League;
    rankingPoints?: number;
}
